spec/setup_test_database.rb

```diff
@@ -0,0 +1,7 @@
+require 'pg'
+
+def setup_test_database
+  p "Setting up Chitter's test database..."
+  connection = PG.connect(dbname: 'chitter_test')
+  connection.exec("TRUNCATE peeps;")
```

Comment:
So I decided to also test my user's sign-up process using a test database so I had to truncate my users table as well. I am assuming you added default test users to your test database and just truncated your peeps table?